src/magnify/apps/core/serializers/rooms.py

```diff
@@ -116,29 +116,37 @@ def to_representation(self, instance):
 
         if not request:
             return output
+
         user = request.user
+        role = instance.get_role(user)
+        is_admin = role in [models.UserRoleChoices.OWNER, models.UserRoleChoices.ADMIN]
```

Comment:
```suggestion
        role = instance.get_role(user)
        is_admin = role in [models.UserRoleChoices.OWNER, models.UserRoleChoices.ADMIN]
```
don't see the point here, when we have a is_administrator method that does exactly the same job

 return self.get_role(user) in [UserRoleChoices.ADMIN, UserRoleChoices.OWNER]

Reply:
True... The point here was to not incur a new db query.

Reply:
then maybe you are missing a static method used by the model as well so you don't repeat the logic of what it means to be an admin

Reply:
Or... rename the variable here to "is_admin_or_owner"? Because is_admin when in fact I'm trying to extend to owner is not clear...

Reply:
I have proposed a refactoring that should please you: https://github.com/openfun/jitsi-magnify/pull/135/commits/b8d2a013555a7b1a4fcbada65737d7c7b1f5380a